lib/mysql2-cs-bind.rb

```diff
@@ -29,17 +29,26 @@ def self.pseudo_bind(sql, values)
 
     while pos = placeholders.pop()
       rawvalue = values.pop()
-      if rawvalue.nil?
-        sql[pos] = 'NULL'
-      elsif rawvalue.respond_to?(:strftime)
-        sql[pos] = "'" + rawvalue.strftime('%Y-%m-%d %H:%M:%S') + "'"
-      elsif rawvalue.is_a?(Array)
-        sql[pos] = rawvalue.map{|v| "'" + Mysql2::Client.escape(v.to_s) + "'" }.join(",")
+      if rawvalue.is_a?(Array)
+        sql[pos] = rawvalue.map{|v| quote(v) }.join(",")
```

Comment:
I've found Array of time objects, true/false, nil was not escaped well.
I fixed together, but I will send a separated PR if you like.